smartcontractkit/external-adapters-js
# [Feat] Composite input parameters passthrough

## Closes 20330 & 20333

## Description
Allows composite adapters to pass input through normalization middlewares to upstream adapters by merging `inputParameter` data schemas.

## Changes

- Add and consume composite input selector
- Change source adapters & examples to use module root exports and export their endpoints

## Steps to Test

1. Run a source adapter (e.g. Kaiko)
2. Run Vesper adapter with Kaiko configured as a data source
3. Send a request to Vesper with a Kaiko specific input parameter (e.g. `base`)
4. Kaiko should receive the input parameter

## Quality Assurance
